doc/concepts/index.rst

```diff
@@ -0,0 +1,35 @@
+.. _concepts:
+
+Concepts
+========
+
+Pachyderm is an enterprise-grade, open source data science platform that
+makes explainable, repeatable, and scalable Machine Learning (ML) and
+Artificial Intelligence (AI) a reality. The Pachyderm platform brings
+together version control for data with the tools to build scalable
+end-to-end ML/AI pipelines while empowering users to develop their
+code in a language, framework, or tool of their choice. Pachyderm
+has been proven to be the ideal foundation for teams looking to
+use ML/AI to solve real-world problems in a reliable way. What
+makes Pachyderm such a natural choice for data science teams is
+that with just one click, a user can know exactly what data was used
```

Comment:
> that with just one click, a user can know exactly what data was used
to train a model

I remember seeing this sentence in Nicks marketing material and I don't really like it. Nothing is happening with "one-click" -- I get the point he's trying to make, but it feels fluffy to me as it probably does to you too. I think just remove the last sentence completely ... end with "reliable way."

Please make a note that after we finish the website copy for these marketing materials, we'll come back and edit this section. For example, IMO, its much too ML/AI focused instead of broader Data Science and Data Engineering, but that's a Nick thing that we'll just copy over to here later.